otelcol coredumps on startup in 0.68.1

I have been tweaking my setup both in the agents & clusterreceiver and although I am using the helm chart, I upgraded the image to 0.68.1.  On startup, just the cluster receiver is core dumping.  The agents are starting up fine.  I am reasonably sure I have something funky in my setup, but that should still not cause a core dump.

Here are the details.

```
2023/01/12 18:48:00 settings.go:310: Set config to [/conf/relay.yaml]
2023/01/12 18:48:00 settings.go:363: Set ballast to 165 MiB
2023/01/12 18:48:00 settings.go:379: Set memory limit to 450 MiB
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x98 pc=0x3e7a335]

goroutine 1 [running]:
github.com/signalfx/splunk-otel-collector/internal/configconverter.RenameK8sTagger.Convert({}, {0xc00120e390?, 0xc00120e360?}, 0xc000681f60)
        github.com/signalfx/splunk-otel-collector/internal/configconverter/k8s_tagger.go:56 +0x3d5
go.opentelemetry.io/collector/confmap.(*Resolver).Resolve(0xc001301b60, {0x5ce3f70, 0xc000076030})
        go.opentelemetry.io/collector/confmap@v0.68.1-0.20221221114823-4cf50d0f0d9d/resolver.go:178 +0x28e
go.opentelemetry.io/collector/otelcol.(*configProvider).Get(0x5cab2e0?, {0x5ce3f70?, 0xc000076030?}, {0xc00049fad0?, 0xc000599ce0?, 0xc000598ff0?, 0xc000be1ef0?})
        go.opentelemetry.io/collector@v0.68.1-0.20221221114823-4cf50d0f0d9d/otelcol/configprovider.go:91 +0x72
go.opentelemetry.io/collector/otelcol.(*Collector).setupConfigurationComponents(0xc00129f788, {0x5ce3f70, 0xc000076030})
        go.opentelemetry.io/collector@v0.68.1-0.20221221114823-4cf50d0f0d9d/otelcol/collector.go:154 +0x6f
go.opentelemetry.io/collector/otelcol.(*Collector).Run(0xc00129f788, {0x5ce3f70, 0xc000076030})
        go.opentelemetry.io/collector@v0.68.1-0.20221221114823-4cf50d0f0d9d/otelcol/collector.go:205 +0x65
go.opentelemetry.io/collector/otelcol.NewCommand.func1(0xc000191200, {0x52323de?, 0x0?, 0x1?})
        go.opentelemetry.io/collector@v0.68.1-0.20221221114823-4cf50d0f0d9d/otelcol/command.go:53 +0x419
github.com/spf13/cobra.(*Command).execute(0xc000191200, {0xc000072050, 0x0, 0x1})
        github.com/spf13/cobra@v1.6.1/command.go:916 +0x862
github.com/spf13/cobra.(*Command).ExecuteC(0xc000191200)
        github.com/spf13/cobra@v1.6.1/command.go:1044 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
        github.com/spf13/cobra@v1.6.1/command.go:968
main.runInteractive({{0xc00049fad0, 0xc000599ce0, 0xc000598ff0, 0xc000be1ef0}, {{0x52319c0, 0x7}, {0x0, 0x0}, {0x5c890a0, 0x7}}, ...})
        github.com/signalfx/splunk-otel-collector/cmd/otelcol/main.go:118 +0x5d
main.run(...)
        github.com/signalfx/splunk-otel-collector/cmd/otelcol/main_others.go:26
main.main()
        github.com/signalfx/splunk-otel-collector/cmd/otelcol/main.go:111 +0x1498

```

--- a/internal/configconverter/k8s_tagger.go
+++ b/internal/configconverter/k8s_tagger.go
@@ -53,8 +53,8 @@ func (RenameK8sTagger) Convert(_ context.Context, in *confmap.Conf) error {
 			found = true
 		} else {
 			if serviceEntryRe.MatchString(k) {
-				kind := reflect.TypeOf(v).Kind()
-				if kind == reflect.Slice {
+				t := reflect.TypeOf(v)
+				if t != nil && t.Kind() == reflect.Slice {
 					if sliceOfInterfaces, ok := v.([]any); ok {
 						for i, val := range sliceOfInterfaces {
 							if strVal, ok := val.(string); ok {

